letsencrypt/auth_handler.py

```diff
@@ -42,7 +42,7 @@ class AuthHandler(object):
         form of :class:`letsencrypt.achallenges.AnnotatedChallenge`
 
     """
-    def __init__(self, dv_auth, cont_auth, acme, account):
+    def __init__(self, dv_auth, cont_auth, acme, account, config=None):
```

Comment:
`config=None` should be changed to just `config` (i.e. without default value) - this class won't work without valid `IConfig` - moreover, you should update the docstring for the class